ffi/src/clipboard/message.rs

```diff
@@ -0,0 +1,65 @@
+#[diplomat::bridge]
+pub mod ffi {
+
+    #[diplomat::opaque]
+    pub struct ClipboardMessage(pub ironrdp::cliprdr::backend::ClipboardMessage);
+
+    impl ClipboardMessage {
+        pub fn get_enum_type(&self) -> ClipboardMessageType {
```

Comment:
naming: get_type

The returned data structure being an "enum" is not relevant

Reply:
GetType is a C# reserved function for all classes.

Reply:
In that case, `get_message_type` may be a better alternative

Reply:
updated